src/Components/DrillListPage.js

```diff
@@ -18,6 +19,9 @@ export const DrillListPage = props => {
   const [data, setData] = useState(drills);
   const [displayFilters, setDisplayFilters] = useState(false);
 
+  const imageMainData = type === DrillTypes.TECHNICAL ? 'nbPlayers' : 'durationInMinutes';
+  const imageMainDataLegend = type === DrillTypes.TECHNICAL ? 'players' : 'min.';
```

Comment:
It looks like it is always technical on my device for technical and fitness drills ... Is it normal ?

Reply:
Yes I made a mistake on the home page when using enums. I fix it

Reply:
Good that I wrote tests on navigation in the other PR :D

Reply:
Ahaha